src/client/components/ui/ErrorMessage/NoResultErrorMessage.tsx

```diff
@@ -1,11 +1,17 @@
 import { ErrorMessageComponent } from '~/client/components/ui/ErrorMessage/ErrorMessageComponent/ErrorMessageComponent';
 
-export function NoResultErrorMessage() {
+interface noResultErrorMessage {
```

Comment:
```suggestion
interface NoResultErrorMessageProps {
```
une interface commence toujours par une majuscule ^^